src/hotspot/share/prims/forte.hpp

```diff
@@ -29,6 +29,7 @@
 
 class Forte : AllStatic {
  public:
+   static bool is_enabled() NOT_JVMTI_RETURN_(false);
```

Comment:
I don't think the rest of this forte code is disabled by JVMTI.

Reply:
If the answer to whether it's enabled is something you want to be fast, and doesn't change, maybe make it check a variable?

Reply:
The code in the `Forte` class, as well as the non-trivial of `AsyncGetCallTrace()` in forte.cpp, are inside `#if INCLUDE_JVMTI`. That's why I use `NOT_JVMTI_RETURN_` for the new function.

Reply:
Ok, thanks for answering my question.